src/pages/Explore_Regions.py

```diff
@@ -119,54 +127,101 @@ def plot_comp(m_route, cause, df):
 
 #  Markdown for the page
 st.header("Explore One Region at a Time")
-st.markdown("In this page we welcome you to explore the data one region at a time, using the filters in the left side menu to filter. Try using the filters and check how the data visualization changes." +
-"Migrations that cross borders and routes include entire families with children. Those routes are so dangerous, and even more for infants." + "On the other hand, others manage to survive the journey, but the fact that they do does not mean that it was without great difficulty and that many others did not.")
+st.markdown("In this page we welcome you to explore the data one region at a time, using the filters in the left side menu. Try selecting a new region to check how the data visualization changes. " +
+            "Migrations that cross borders and routes often include families with children. While extremely dangerous, they are even moreso for infants." + " For those that do survive the journey, it is not without great difficulty and loss.")
+
+st.markdown("The following is a table of all included migration routes paired with the total number of recorded dead and missing people from that route, since 2014.")
+dff = df.groupby(['Migration route'])[
+    ['Total Number of Dead and Missing']].sum().sort_values('Total Number of Dead and Missing', ascending=False).reset_index()
+st.dataframe(dff)
+
 st.sidebar.write("Data Filters")
+
 route_input = [st.sidebar.selectbox(
     'Migration Route', df['Migration route'].unique().tolist())]
+st.header('Exploring the '+route_input[0])
```

Comment:
Added in this header and subheaders/markdown throughout to explain what it is we're showing